spec/factories/address.rb

```diff
@@ -2,8 +2,24 @@
 
 FactoryBot.define do
   address_types = WasteExemptionsEngine::Address.address_types
+  modes = WasteExemptionsEngine::Address.modes
 
   factory :address, class: WasteExemptionsEngine::Address do
+    trait :manual do
+      mode { modes[:manual] }
+      premises { "#{address_type} Address Premises" }
+      street_address { "#{address_type} 34 Honey road" }
+      locality { "#{address_type} Avon" }
+      city { "#{address_type} Bristol" }
+      postcode { "#{address_type} BS1 5AH" }
```

Comment:
Since the PDF contains all the addresses added in the registration phase, and I have the need to check that a particular address has been added for a specific type, this will make sure that when I check for the "site manual address" (https://github.com/DEFRA/waste-exemptions-engine/pull/134/files#diff-192d7a07fa048762e8a972439ffdd1b7R42) the test doesn't pass just because other addresses in the PDF are equal to this. Open to suggestions if this is not right :)